project-base/storefront/components/Pages/Order/ContactInformation/ContactInformationContent.tsx

```diff
@@ -69,37 +69,37 @@ export const ContactInformationWrapper: FC = () => {
         setOrderCreating(true);
 
         let deliveryInfo = {
-            deliveryFirstName: formValues.differentDeliveryAddress ? formValues.deliveryFirstName : '',
```

Comment:
please, add the information about the renaming into the "Storefront" section of the upgrade notes as well :pray:

Reply:
ok, added